sw/airborne/math/pprz_stat.h

```diff
@@ -33,47 +33,13 @@ extern "C" {
 
 #include "std.h"
 
-/** Compute the variance of an array of values (float).
- *  The variance is a measure of how far a set of numbers is spread out
- *  V(X) = E[(X-E[X])^2] = E[X^2] - E[X]^2
- *  where E[X] is the expected value of X
- *
- *  @param array pointer to an array of float
- *  @param nb numbre of values in the array, must be >0
- *  @return variance
- */
-static inline float variance_float(float *array, int nb)
-{
-  float me = 0.;
-  float see = 0.;
-  for (int i = 0; i < nb; i++) {
-    me += array[i];
-    see += array[i] * array[i];
-  }
-  me /= nb;
-  return (see / nb - me * me);
-}
+extern int32_t mean_i(int32_t *array, uint32_t n_elements);
```

Comment:
I would keep the doxygen comments for the header as well, although I don't know how doxygen is extracting the information